espaloma/nn/readout/janossy.py

```diff
@@ -197,6 +198,9 @@ def __init__(
         # bookkeeping
         self.out_features = out_features
         self.levels = ["n4_improper"]
+        if improper_def not in ('smirnoff', 'espaloma'):
```

Comment:
might be nice moving this to another object called `JanossyPoolingWithSmirnoffImproper` or whatever